### DISCUSSION TITLE
Current state of v2

### DISCUSSION THREAD

**Discussion by renepardon** [thedevdojo/voyager#5678] [General]
Hey guys,

what’s the current state of v2? Is there a roadmap available? What is missing for v2? Something I could maybe help with?

**Comment by edgar4all** [comment]
Looking forward for the V2 too...

**Comment by emptynick** [comment]
The development is happening here: https://github.com/voyager-admin/voyager
Unfortunately, we don't have the time to develop that much at the moment.
Nonetheless, things are looking good already. Some major features are still missing and some (bigger) bugs might hit you.

**Reply by Marcus5114** [reply]
Bootstrap 5 maybe??
